src/libs/SidebarUtils.ts

```diff
@@ -88,13 +88,17 @@ function getOrderedReportIDs(
             return;
         }
         const reportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`] ?? {};
+        const parentReportAction =
```

Comment:
```suggestion
        const parentReportAction = ReportActionsUtils.getReportAction(report?.parentReportID ?? '-1', report?.parentReportActionID ?? '-1');
```

Reply:
Hmm, it's basically the same as deprecated getParentReportAction which I guess gonna be deprecated in the future, but I updated it to not block the merge.